src/scope/component-remove.js

```diff
@@ -0,0 +1,85 @@
+import R from 'ramda';
+import chalk from 'chalk';
+import { BitIds } from '../bit-id';
+
+export class RemovedObjects {
+  removedComponentIds: BitIds;
+  missingComponents: BitIds;
+  dependentBits: Object;
+  removedDependencies: BitIds;
+  constructor(
+    bitIds: BitIds = [],
+    missingComponents: BitIds = [],
+    removedDependencies: BitIds = [],
+    dependentBits: Object = {}
+  ) {
+    this.removedComponentIds = bitIds;
+    this.missingComponents = missingComponents;
+    this.dependentBits = dependentBits;
+    this.removedDependencies = removedDependencies;
+  }
+
+  paintMissingComponents = () =>
```

Comment:
I really don't like all the paintings here.. it's class under the scope. it shouldn't be aware of painting at all.
Create a function in the CLI/templates which knows to get a RemovedObjects instance and print it.